hidden-posts.php

```diff
@@ -20,177 +22,205 @@
  */
 class Hidden_Posts {
 
-    const META_KEY = 'hidden-posts';
-    const NONCE_KEY = 'hidden-posts-nonce';
-
-    /**
-     * Maximum number of posts to store in the hidden array
-     */
-    const LIMIT = 100;
-
-    function __construct() {
-        add_action( 'save_post', array( $this, 'save_meta' ) );
-        add_action( 'add_meta_boxes', array( $this, 'add_metabox'  ) );
-        add_action( 'pre_get_posts', array( $this, 'pre_get_posts' ) );
-        add_filter( 'manage_posts_columns', array( $this, 'custom_column_title' ) );
-        add_filter( 'manage_posts_custom_column', array( $this, 'custom_column_data'), 10, 2 );
-        add_action( 'admin_head', array( $this, 'custom_column_style' ) );
-    }
-
-    /**
-     * Hide the posts in the hidden array on the homepage
-     */
-    function pre_get_posts( $query ) {
-        if ( is_admin() || ! $query->is_main_query() )
-            return;
-
-        if ( apply_filters( 'hidden_posts_show_posts', is_single() ) )
-            return;
+	const META_KEY  = 'hidden-posts';
+	const NONCE_KEY = 'hidden-posts-nonce';
+
+	/**
+	 * Maximum number of posts to store in the hidden array
+	 */
+	const LIMIT = 100;
+
+	/**
+	 * Get hooked in!
+	 */
+	public function __construct() {
+		add_action( 'save_post', array( $this, 'save_meta' ) );
+		add_action( 'add_meta_boxes', array( $this, 'add_metabox' ) );
+		add_action( 'pre_get_posts', array( $this, 'pre_get_posts' ) );
+		add_filter( 'manage_posts_columns', array( $this, 'custom_column_title' ) );
+		add_action( 'manage_posts_custom_column', array( $this, 'custom_column_data' ), 10, 2 );
+		add_action( 'admin_head', array( $this, 'custom_column_style' ) );
+	}
+
+	/**
+	 * Hide the posts in the hidden array on the homepage
```

Comment:
According to https://docs.phpdoc.org/3.0/guide/references/phpdoc/basic-syntax.html#basic-syntax, summaries such as _Hide the posts ..._ should end with a full stop to prevent that descriptions are treated as part of the summary.